tests/integration/test_raw_persistence.py

```
@@ -0,0 +1,305 @@
+        
+        # Verify raw data was persisted
+        raw_record = test_db_session.query(RawIngestion).filter_by(
+            ingestion_id=ingestion_id
+        ).first()
+        
+        assert raw_record is not None
+        assert raw_record.source_type == "text"
+        assert raw_record.content_type == "text"
+        assert raw_record.raw_payload["content"] == payload["content"]
+        assert raw_record.metadata["tags"] == ["test", "python"]
+        assert raw_record.processed == False
```

Ruff (0.14.11)

Avoid equality comparisons to `False`; use `not raw_record.processed:` for false checks

Replace with `not raw_record.processed`

(E712)